src/ray/pubsub/mock_pubsub.h

```diff
@@ -24,17 +24,29 @@ namespace mock_pubsub {
 
 class MockSubscriber : public pubsub::SubscriberInterface {
  public:
-  MOCK_METHOD6(Subscribe,
-               void(std::unique_ptr<rpc::SubMessage> sub_message,
+  MOCK_METHOD7(Subscribe,
+               bool(std::unique_ptr<rpc::SubMessage> sub_message,
                     const rpc::ChannelType channel_type,
                     const rpc::Address &owner_address, const std::string &key_id,
-                    pubsub::SubscriptionCallback subscription_callback,
+                    pubsub::SubscribeDoneCallback subscribe_done_callback,
+                    pubsub::SubscriptionItemCallback subscription_callback,
+                    pubsub::SubscriptionFailureCallback subscription_failure_callback));
+
+  MOCK_METHOD6(SubscribeAll,
+               bool(std::unique_ptr<rpc::SubMessage> sub_message,
+                    const rpc::ChannelType channel_type,
+                    const rpc::Address &owner_address,
+                    pubsub::SubscribeDoneCallback subscribe_done_callback,
+                    pubsub::SubscriptionItemCallback subscription_callback,
                     pubsub::SubscriptionFailureCallback subscription_failure_callback));
 
   MOCK_METHOD3(Unsubscribe,
                bool(const rpc::ChannelType channel_type,
                     const rpc::Address &publisher_address, const std::string &key_id));
 
+  MOCK_METHOD2(Unsubscribe, bool(const rpc::ChannelType channel_type,
+                                 const rpc::Address &publisher_address));
```

Comment:
I think we can actually delete this file cuz we have a mock folder

Reply:
Added a TODO to move this to `mock/ray/pubsub`. This file has a number of usages so probably in another PR.